python/aws_lambda_powertools/logging/logger.py

```diff
@@ -26,13 +29,17 @@ def logger_setup(service: str = "service_undefined", level: str = "INFO", **kwar
         service name
     LOG_LEVEL: str
         logging level (e.g. INFO, DEBUG)
+    POWERTOOLS_LOGGER_SAMPLE_RATE: str
```

Comment:
I think you meant float in type annotation here - `POWERTOOLS_LOGGER_SAMPLE_RATE: float`